topics-and-advance-readings/README.md

```diff
@@ -76,6 +76,11 @@ really, read as much as you can!)
   Can we build a compelling Self-Issued OpenID Connect Provider today.
   * #authentication #web #oidc
 
+]Digital Wallets: Interoperability support for multiple data hubs, data services and portability](digital-wallets-interoperability-support-for-multiple-data-hubs-data-services-and-portability.md)
```

Comment:
That first bracket should be reversed...

Reply:
fixed